data/tooling-data.yaml

```diff
@@ -3034,3 +3034,26 @@
   source: 'https://github.com/andreineculau/json-schema-random'
   license: 'Apache-2.0'
   status: 'obsolete'
+
+- name: Jaronuinga
+  description: >
+    Jaronuinga is a simple JSON Schema validator written in Java. It is based on JSONP 1.1 and has no other dependencies.
+    The library's primary purpose is to intercept the parsing process for additional custom analysis (e.g., verifying "foreign keys" on data submission).
+    It may also be used for schema analysis via `getParent()` and `getChildren()` methods.
+  toolingTypes: ['validator']
+  languages: ['Java 17']
```

Comment:
Just use "Java". If the Java version is important, you can put it in the description.

Reply:
done